### PR TITLE
test

### PR DESCRIPTION
**Pull request by GeckoJnlx3**
hi

### CODE REVIEW

File: divide.c [hunk 1]
```diff
@@ -0,0 +1,5 @@
+int divide_num(int x, int y){
+	int ans;
+	div = x/y;
```
Comment (dhannn) [inline divide.c:3]: BUG: you did not define div and you didn't use ans

Comment (dhannn) [reply divide.c:4]: ur talking your team lead like that????? 😠😠

File: divide.c [hunk 2]
```diff
@@ -0,0 +1,6 @@
+int divide_num(int x, int y)
```
Comment (dhannn) [inline divide.c:1]: Edge case: if y is 0, machine goes brrrrrr

**Review by dhannn — CHANGES_REQUESTED** [review]
WOI MAY MALI KA PA AYUSIN MO

File: divide.c [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+{
+	int ans;
+	ans = x/y;
```
Comment (dhannn) [inline divide.c:4]: add space between binary operators

Comment (dhannn) [reply divide.c:4]: for readability

**Review by dhannn — CHANGES_REQUESTED** [review]
had some problems with styles

File: divide.c (hunk 3 above)
Comment (gwnrocs) [inline divide.c]: bakit walang space hays UR FIRED

File: divide.c (hunk 1 above)
Comment (GeckoJnlx3) [reply divide.c]: cant fire me if i leave the team first!!!!!!!!

File: divide.c (hunk 3 above)
Comment (GeckoJnlx3) [reply divide.c:15]: ok wait this one is agreeable

File: divide.c [hunk 4]
```diff
@@ -0,0 +1,18 @@
+int check_divisor(int y)
+{
+	while (y == 0)
+	{
```
Comment (dhannn) [inline divide.c:4]: brace on same line

**Review by dhannn — CHANGES_REQUESTED** [review]
STYLE CONVENTION

File: divide.c (hunk 4 above)
Comment (GeckoJnlx3) [reply divide.c:4]: BUT EARLIER NOT SAME LINE??????????? ok fine

File: divide.c [hunk 5]
```diff
@@ -0,0 +1,17 @@
+int check_divisor(int y)
+{
+	while (y == 0){
```
Comment (dhannn) [inline divide.c:3]: ADD SPACE BETWEEN CLOSING PARANTHESES AND OPENING BRACES/REMOVE SPACE BETWEEN WHILE AND OPENING PARANTHESIS 

`while(y == 0) {`

**Review by dhannn — CHANGES_REQUESTED** [review]
kun bin shawn

File: divide.c [hunk 6]
```diff
@@ -1,7 +1,6 @@
 int check_divisor(int y)
```
Comment (gwnrocs) [inline divide.c]: this function is really not the vibes... it's just not giving.. you get? like . . . it's not... it's not the vibes for real. i'll be forwarding this to our section manager. expect to see point deductions in your monthly evaluation.

Comment (dhannn) [reply divide.c]: TRULY

**Pull request merged**